puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/rubygemRPMNightlyRelease.groovy

```diff
@@ -0,0 +1,57 @@
+/*
+ * this pipeline assumes you've set a handful of groovy variables
+ *
+ * owner_repo
+ * branch
+ * gemspec
+ * package_name
+ */
+
+def obalExtraVars = [
+    'build_package_tito_releaser_args': [
+        "--arg jenkins_job=${env.JOB_NAME}",
+        "--arg jenkins_job_id=${env.BUILD_ID}"
+    ],
+    'releasers': [
+        'koji-katello-jenkins'
```

Comment:
This could be good enough for the first version since we're only on katello anyway, but perhaps this should also be a variable?

Reply:
yeah it could be moved into a variable in the future